electron/gog/games.ts

```diff
@@ -0,0 +1,267 @@
+          install_size: prettyBytes(installInfo.manifest.disk_size),
+          is_dlc: false,
+          version: installInfo.game.version,
+          appName: this.appName,
+          installedWithDLCs: installDlcs,
+          language: 'en-US'
+        }
+        const array: Array<InstalledInfo> =
+          (installedGamesStore.get('installed') as Array<InstalledInfo>) || []
+        array.push(installedData)
+        installedGamesStore.set('installed', array)
+        GOGLibrary.get().refreshInstalled()
+        return { status: 'done' }
+      })
+      .catch(() => {
+        logInfo('Installaton canceled', LogPrefix.Gog)
+        return { status: 'error' }
+      })
+  }
+  addShortcuts(): Promise<void> {
+    throw new Error('Method not implemented.')
+  }
+  removeShortcuts(): Promise<void> {
+    throw new Error('Method not implemented')
+  }
+  launch(launchArguments?: string): Promise<ExecResult | LaunchResult> {
+    return launch(this.appName, launchArguments, 'gog')
+  }
+  public async moveInstall(newInstallPath: string): Promise<string> {
+    const {
+      install: { install_path },
+      title
+    } = await this.getGameInfo()
+
+    if (isWindows) {
+      newInstallPath += '\\' + install_path.split('\\').slice(-1)[0]
+    } else {
+      newInstallPath += '/' + install_path.split('/').slice(-1)[0]
+    }
+
+    logInfo(`Moving ${title} to ${newInstallPath}`, LogPrefix.Gog)
+    await execAsync(`mv -f '${install_path}' '${newInstallPath}'`, execOptions)
+      .then(() => {
+        GOGLibrary.get().changeGameInstallPath(this.appName, newInstallPath)
+        logInfo(`Finished Moving ${title}`, LogPrefix.Gog)
+      })
+      .catch((error) => logError(`${error}`, LogPrefix.Gog))
+    return newInstallPath
+  }
+  /**
+   * Literally installing game, since gogdl verifies files at runtime
+   */
+  public async repair(): Promise<ExecResult> {
+    // const { maxWorkers } = await GlobalConfig.get().getSettings()
+    // const workers = maxWorkers === 0 ? '' : `--max-workers ${maxWorkers}`
+    const gameData = GOGLibrary.get().getGameInfo(this.appName)
+
+    const withDlcs = gameData.install.installedWithDLCs
+      ? '--with-dlcs'
+      : '--skip-dlcs'
+    if (GOGUser.isTokenExpired()) await GOGUser.refreshToken()
+    const credentials = configStore.get('credentials') as GOGLoginData
+
+    const installPlatform = gameData.install.platform
+    // In the future we need to add Language select option
+    const command = `${gogdlBin} repair ${
+      this.appName
+    } --platform ${installPlatform} --path="${
+      gameData.install.install_path
+    }" --token="${credentials.access_token}" ${withDlcs} --lang="${
+      gameData.install.language || 'en-US'
+    }"`
+    logInfo([`Installing ${this.appName} with:`, command], LogPrefix.Gog)
+
+    return execAsync(command, execOptions)
+      .then((v) => v)
+      .catch((error) => {
+        logError(error, LogPrefix.Gog)
```

Comment:
Needed else convert2String() error will be triggered
```suggestion
        logError(`${error}`, LogPrefix.Gog)
```